google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -141,6 +141,10 @@ def time_partitioning_type
         # Sets the partitioning for the table. See [Partitioned Tables
         # ](https://cloud.google.com/bigquery/docs/partitioned-tables).
         #
+        # You can only set partitioning when creating a table as in
+        # the example below. BigQuery does not allow you to change partiting
```

Comment:
I really seem to have trouble typing that word. I'll fix it. :/

I went with this statement rather than what you suggest (which was my first pass) because it also does not allow you to change a partitioned table to an unpartitioned one. In that case the error message is:

> Google::Cloud::InvalidArgumentError: invalid: For time based partitioning, only the partition expiration time can be changed.

But I can still change the statement if that's what you want to say.